autoscheduler/scraper/migrations/0006_auto_20201029_1643.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.6 on 2020-10-29 21:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scraper', '0005_auto_20201026_1519'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='section',
+            name='instructional_method',
+            preserve_default=False,

```

Comment:
Can you delete this file and re-run `makemigrations`? When I did it this `preserve_default=False` line gets deleted